src/features/error-highlighting.ts

```diff
@@ -128,10 +128,12 @@ const Elm = {
     const promise: Promise<ParsedError | undefined> =
       new Promise((resolve, reject) => {
         const isWindows = process.platform === 'win32'
+        const compiler = vscode.workspace.getConfiguration('elmLand').compilerFilepath;
```

Comment:
Wasn't sure if fetching the configuration on every compile has a significant cost, so I didn't bother threading it down from higher up, but happy to do so if you think it would have an impact.